.code-samples.meilisearch.yaml

```diff
@@ -368,3 +368,7 @@ faceted_search_walkthrough_facets_distribution_1: |-
   index.search('Batman', {
     facetsDistribution: ['genres']
   })
+post_dump_1: |-
+  client.create_dump
+get_dump_status_1: |-
+  client.get_dump_status(20201006-053243949)
```

Comment:
Could you add a new line at the end of this file please? :)

Reply:
To avoid this kind of error, you can download the editorconfig plugin, depending on your text editor. After downloading it, every time you will save your file, it will automatically apply the basic style rules to your file (removing trailing whitespaces, adding a new line at the end of file...).
See more about editor config: https://editorconfig.org/#overview
And the editorconfig file of this repo: https://github.com/meilisearch/meilisearch-ruby/blob/master/.editorconfig

Reply:
Ok, thank you. I thought rubocop took care of that. I've installed editorconfig in my editor :)

Reply:
I thought the same when I started this project, but unfortunately, no.